consensus/complete_test.go

```diff
@@ -35,12 +35,12 @@ func currentPathCheck(t *testing.T, s *State) {
 // Then the state moves forwards to the initial starting place and verifies
 // that the state hash is the same.
 func rewindApplyCheck(t *testing.T, s *State) {
-	stateHash := s.stateHash()
+	stateHash := s.Hash()
```

Comment:
ah. I was going to rename this to `ConsensusSetHash`, because you're not hashing the whole state you're only hashing the consensus set.